secure-mobile.md

```diff
@@ -40,6 +40,10 @@ Newer iOS and Android devices feature multiple methods for locking your device,
 
 [Follow the official Google guide](https://support.google.com/android-one/answer/6174402) on enabling the SIM card lock. _Note: this may not be necessary for services like Project Fi, and the SIM lock option may be missing from the device settings._
 
+### Avoid Apps from Unknown Sources
```

Comment:
Remove this section.

Reply:
Did you want to put info like that in another section or you don't want that mentioned at all? Seems like good advice.

Reply:
Disregard I saw your comment on the closed PR. Should we perhaps indicate it could come from a store such as aurora or fdroid, which actually wouldn't be linked to your google account and wouldn't require you to make an account. I do that for myself. Overall I think making sure an app you download is verifiable in some way or at least as a reputation whether it's from a store or somewhere else.